prepare_mkl.sh

```diff
@@ -68,8 +68,8 @@ if [ ! -d "$HOME_MKL" ]; then
 fi
 MXNET_ROOT=`dirname $0`
 OMP=0 
-VERSION_MATCH=20161123
-ARCHIVE_BASENAME=mklml_lnx_2017.0.2.20161122.tgz
+VERSION_MATCH=20170210
+ARCHIVE_BASENAME=mklml_lnx_2017.0.2.20170209.tgz
```

Comment:
Every time mkldnn updates the user need to reset LD_LIBRARY_PATH. Its a lot of confusion.
 Please change it to a permanat address by making a symbolic link mxnet/mklmkl_lnx/